libraries/rush-lib/src/cli/test/pluginWithConflictRebuildCommandRepo/common/autoinstallers/plugins/rush-plugins/rush-rebuild-command-plugin/rush-rebuild-command-plugin/command-line.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "../../../../../../../../../schemas/command-line.schema.json",
+  "commands": [
```

Comment:
```ts
// If no command-line.json file defines "rush build" and "rush rebuild", then normally Rush's default 
// definitions are used as if they were defined in  common/config/rush/command-line.json, 
// and therefore the corresponding phases must be in that file as well.  But because the line below
// customizes "rush rebuild" in this plugin, then the defaults are applied here, and as a result the 
// corresponding phases must also be defined by this plugin.
```

(Although this is just a test file, explaining this fact really helps to understand what we're testing.)